packages/shared-components/src/utils/tooltip-util.ts

```diff
@@ -1,11 +1,63 @@
+import {OntoTooltipConfiguration} from "../components/onto-tooltip/models/onto-tooltip-configuration";
+import tippy, {Instance} from "tippy.js";
+import {OntoTooltipPlacement} from "../components/onto-tooltip/models/onto-tooltip-placement";
+import {HTMLElementWithTooltip} from "../components/onto-tooltip/models/html-element-with-tooltip";
+
 export class TooltipUtil {
+  private static readonly ATTR_CONTENT = 'tooltip-content';
+  private static readonly ATTR_THEME = 'tooltip-theme';
+  private static readonly ATTR_PLACEMENT = 'tooltip-placement';
+  private static readonly ATTR_TRIGGER = 'tooltip-trigger';
+  private static readonly ATTR_APPEND_TO = 'tooltip-append-to';
+
+  /**
+   * Returns the Tippy instance associated with an element, if it exists.
+   */
+  static getTooltipInstance(element: HTMLElement): import('tippy.js').Instance | undefined {
```

Comment:
I've never seen that one before. Does it not work if the import is in the import section on the top?